guide/matlab-observation-space.rst

```diff
@@ -1,9 +1,27 @@
-#####################################
-MATLAB® observation space diagnostics
-#####################################
+####################################
+MATLAB observation space diagnostics
+####################################
 
-The observation-space functions are in the ``$DARTROOT/diagnostics/matlab``
-directory. Once you have processed the ``obs_seq.final`` files into a single
+Configuring MATLAB
+==================
+
+DART uses MATLAB's own netCDF reading and writing capability and does not use
+any MATLAB or third-party toolboxes. 
+
+To allow your environment to seamlessly use the DART MATLAB functions, your
+MATLAB path must be set to include two of the directories in the DART
+repository. In the MATLAB command prompt enter the following, using the real
+path to your DART installation:
+
+.. code-block::
+
+   addpath('DART/diagnostics/matlab','-BEGIN')
+   addpath('DART/guide/DART_LAB/matlab','-BEGIN')
+
```

Comment:
```It's very convenient to put these commands in your ~/matlab/startup.m so they get run every time MATLAB® starts up. DART provides an example diagnostics/matlab/startup.m that you can use. It is internally documented.```

Reply:
I think we want to add this too -- right?

Reply:
You're right. I added those instructions.